src/scrape_up/Youtube/yt_scraper.py

```diff
@@ -0,0 +1,21 @@
+import requests
+from bs4 import BeautifulSoup
+
+def fetch_html(url):
+    # Make an HTTP GET request to the given URL and return the HTML content
+    # Use the requests library
+
+def parse_html(html_content):
+    # Parse the HTML content using BeautifulSoup and extract the desired data
+    # Use the BeautifulSoup library
+
+def scrape_youtube(url):
+    # Main scraping function that combines the fetching and parsing logic
+    html_content = fetch_html(url)
+    data = parse_html(html_content)
+    return data
+
+# Example usage
+video_url = "https://www.youtube.com/watch?v=YOUR_VIDEO_ID"
+scraped_data = scrape_youtube(video_url)
```

Comment:
What does this method do? Please upload the screenshot of the output.

Reply:
the method scrape_youtube() is called with a video_url parameter, which is a string representing the URL of a YouTube video. @nikhil25803